diamm/helpers/object_pagination.py

```diff
@@ -1,9 +1,9 @@
-from collections import OrderedDict
 import re
-
+from collections import OrderedDict
```

Comment:
What's our style guideline for imports? I was using [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)